l10n_us_form_1099/models/res_partner.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Open Source Integrators <https://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is1099 = fields.Boolean('Is a 1099')
```

Comment:
Please use snake case var names- `is_1099` & align the onchange `_on_change_is_1099`